tests/RealMeServiceTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+class RealMeServiceTest extends SapphireTest
+{
+    private $pathForTempCertificate;
+
+    public function testGetCertificateContents()
+    {
+        $this->pathForTempCertificate = ASSETS_PATH . '/tmpcert.pem';
+
+        // Test standard certificate
+        $contents = <<<EOF
```

Comment:
I'd probably keep these test files as actual files in the test dir then copy them to where they need to go. Leaving them inline like this feels a bit fragile